ecommerce/src/SliderData.js

```diff
@@ -0,0 +1,17 @@
+export const SliderData = [
+    {
```

Comment:
SliderData file contains pure js code, you can rename it to slider-data.js
and constant SliderData should be renamed to sliderData